src/Microsoft.ML.Data/EntryPoints/ScoreColumnSelector.cs

```diff
@@ -99,10 +98,10 @@ public sealed class RenameBinaryPredictionScoreColumnsInput : TransformInputBase
                         }
                         var source = input.Data.Schema.GetColumnName(i);
                         var name = source + "." + positiveClass;
-                        copyCols.Add(new CopyColumnsTransform.Column() { Name = name, Source = source });
+                        copyCols.Add(new CopyColumnsTransformer.Column() { Name = name, Source = source });
                     }
 
-                    var copyColumn = new CopyColumnsTransform(env, new CopyColumnsTransform.Arguments() { Column = copyCols.ToArray() }, input.Data);
+                    var copyColumn = CopyColumnsTransformer.Create(env, new CopyColumnsTransformer.Arguments() { Column = copyCols.ToArray() }, input.Data);
```

Comment:
>Create [](start = 60, length = 6)

Let's try to refactor this method to avoid calling `Create`.
Namely, collect these 'source, target' pairs, call the ctor to `CopyColumnsTransformer` and then `Transform(input.Data)`

Generally, let's try to eliminate args and calls to `Create`, since we want them to be contained in cmdline